src/components/Line/Line.jsx

```diff
@@ -75,6 +83,7 @@ const Line = createClass({
 				}}
```

Comment:
ah, yes, you're right, i forgot these aren't new files. There are no new component files in this PR?